### PR TITLE
Add some processing to the WB32 MCU series.

### PR DESCRIPTION
**Pull request by itarze**
1. Fix the problem that the new version cannot remotely wake up.
2. Added the implementation of embedded flash.

### CODE REVIEW

File: os/hal/ports/WB32/LLD/USBv1/hal_usb_lld.h [hunk 1]
```diff
@@ -472,6 +472,7 @@ struct USBDriver {
     WB32_USB->POWER |= USB_POWER_RESUME;                                    \
     osalThreadSleepMilliseconds(WB32_USB_HOST_WAKEUP_DURATION);             \
     WB32_USB->POWER &= ~USB_POWER_RESUME;                                   \
+    _usb_wakeup(usbp);                                                      \
```
Comment (fpoussin) [inline os/hal/ports/WB32/LLD/USBv1/hal_usb_lld.h:475]: Could you explain this change?

Comment (itarze) [reply os/hal/ports/WB32/LLD/USBv1/hal_usb_lld.h:475]: When remote wakeup is triggered, MCU does not trigger the resume interrupt and needs to be invoked manually.

File: os/hal/ports/WB32/LLD/GPIOv1/hal_pal_lld.c [hunk 2]
```diff
@@ -96,6 +96,12 @@ void _pal_lld_setgroupmode(ioportid_t port, ioportmask_t mask, iomode_t mode) {
   uint32_t altr = (mode & PAL_WB32_ALTERNATE_MASK) >> 7;
   uint32_t current = (mode & PAL_WB32_CURRENT_MASK) >> 11;
 
+#ifdef SPI_SCK_PIN
```
Comment (zvecr) [inline os/hal/ports/WB32/LLD/GPIOv1/hal_pal_lld.c]: It should be noted that this is a QMK specific define, and would hint that the solution its trying to provide is incorrectly coupled.

Comment (itarze) [reply os/hal/ports/WB32/LLD/GPIOv1/hal_pal_lld.c]: Okay, can you tell me what the right thing to do is？

Comment (itarze) [reply os/hal/ports/WB32/LLD/GPIOv1/hal_pal_lld.c]: The problem has been dealt with qmk/qmk_firmware#17650.

**Comment by itarze** [comment]
Can this PR be merged?

**Review by fpoussin — APPROVED** [review]

**Comment by fpoussin** [comment]
All good, thanks!

**Pull request merged**